.eslintrc

```diff
@@ -0,0 +1,198 @@
+{
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module"
+  },
```

Comment:
`env` was removed completely?  is `"node": true` the default?

Reply:
I have kept `"node": true` as it is and have removed `"browser":true`.
Is it ok if I removed browser or shall I make it `"browser": false` ?

Reply:
you should look up what the default value is in [ESLint](https://eslint.org/)'s documentation.  If it's `false`, then that's what we want since this code is not intended to run in the context of a browser.